releasenotes/notes/create-datadog-connector-component-in-otelagent-567d26fefad2e083.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Fork datadogconnector component into OTel agent from the opentelemetry-collector-contrib repo (https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/datadogconnector). In this fork, tagger is called during APM metrics computation, so internal users should now see container tags on their APM metrics.
```

Comment:
```suggestion
    Forks the datadogconnector component into the OTel agent from the opentelemetry-collector-contrib repo (https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/datadogconnector). In this fork, a tagger is called during APM metrics computation, so internal users can now see container tags on their APM metrics.
```

Reply:
It's not clear what's meant by `internal users`. Would it be possible to clarify this?